src/main/kotlin/solve/settings/visualization/fields/view/VisualizationSettingsLayerCell.kt

```diff
@@ -87,6 +88,15 @@ class VisualizationSettingsLayerCell(
         return snapshotNode.createSnapshot()
     }
 
+    override fun isAbleToDropItem(thisItem: LayerSettings, droppedItem: LayerSettings): Boolean {
```

Comment:
what is done here?

Reply:
It checks if swapping items are both planes or both are not planes. This is necessary because we want the plane layers to be in the background so they can't be swapped with point or line layers